sdk/appconfiguration/app-configuration/samples/v1/javascript/helloworldWithLabels.js

```diff
@@ -1,17 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-// This sample builds on concepts in helloworld.ts and shows you how
-// to use labels. Labels allow you to add an extra dimension for your
-// setting and gives you a simple way to create conventions for environments.
-//
-// For additional information about how labels work see:
-// https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-key-value#label-keys
-
+/**
+ * @summary This sample builds on concepts in helloworld.ts and shows you how to use labels. Labels allow you to add an extra dimension for your
```

Comment:
FWIW it's not required that all text that appears in this comment be under the `@summary` tag. Only the part you wish to show up in the README tables. It's fine to have a paragraph (or an essay, if you really want), with only a line or two of `@summary` at the end for the sake of the README.

Reply:
simplified